en/includes/guides/users/attributes/manage-attributes.md

```diff
@@ -93,3 +93,194 @@ To update the properties of a user attribute:
 6. Go to the **Additional Properties** tab and add additional properties that can be used when writing an extension.
 
     ![Edit additional properties]({{base_path}}/assets/img/guides/organization/attributes/edit-attributes-additional-properties.png){: width="500" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+{{ user_attribute_change_verification }}
+
+{% if product_name == "WSO2 Identity Server" %}
+### Try it out
+
+You can try out the follwoing flows using the provided cURLs:
+
+- Update mobile number
+
+    === "Request"
+        ```
+        curl -v -k --user [username]:[password] -X PATCH
+        -d '{
+            "schemas":[],
+            "Operations":[{"op":[operation], 
+            "value":{[attributeName]:[attribute value]}}]
+        }' 
+        --header "Content-Type:application/json" 
+        https://localhost:9443/scim2/Users/[user ID]
+        ```
+
+    === "Sample Request"
+        ```
+        curl -v -k --user bob:pass123 -X PATCH 
```

Comment:
Let's use admin:admin credentials in this case as well

Reply:
https://github.com/wso2/docs-is/pull/4545/commits/e4f326e7fa597eb029baf9e557e7049ee6ec0a2d